src/frontend/src/views/AgreeTerms/style.tsx

```diff
@@ -20,3 +20,13 @@ export const Details = styled.p`
 export const SpacedBold = styled.b`
   padding: 0 0.3em;
 `;
+
+export const StyledIcon = styled(Icon)`
+  ${(props) => {
+    const colors = getColors(props);
+
+    return `
+      fill: ${colors?.gray[500]};
```

Comment:
@Janeece I'm currently using gray[500] for the spinner, should it be lighter?

Reply:
SDS spec says it should be gray[400] because its a disabled state button with an icon. We should go with that to be consistent.